dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateConstants.java

```diff
@@ -0,0 +1,8 @@
+package com.dotmarketing.portlets.templates.business;
+
+public class TemplateConstants {
+
+    public static String TEMPLATE_DELETE_ERROR = "message.template.full_delete.error";
+    public static int TEMPLATE_DEPENDENCY_SEARCH_LIMIT = 10;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "TEMPLATE_DEPENDENCY_SEARCH_LIMIT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static TEMPLATE_DEPENDENCY_SEARCH_LIMIT" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make TEMPLATE_DEPENDENCY_SEARCH_LIMIT a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)